cli/src/main.rs

```diff
@@ -235,7 +237,13 @@ async fn run(cli: Cli) -> anyhow::Result<i32> {
                     print_bep_results(&bep_result);
                     bep_result.uncached_xml_files()
                 }
-                None => junit_paths,
+                None => junit_paths
+                    .into_iter()
+                    .map(|p| JunitPathWrapper {
+                        junit_path: p,
+                        status: None,
+                    })
+                    .collect(),
```

Comment:
We should probably implement `From<String>` for `JunitPathWrapper` since we repeat that often